source/features/useful-not-found-page.tsx

```diff
@@ -223,6 +249,15 @@ void features.add(import.meta.url, 	{
 		pageDetect.isPRCommit404,
 	],
 	init: onetime(initPRCommit),
+}, {
+	asLongAs: [
+		pageDetect.isRepoFile404,
```

Comment:
This already includes isSingleFile and isRepoTree, right? We don't need the extra `include`